### ISSUE TITLE
Require the view field in the bulk (CSV) import to avoid the importer failing the import

### ISSUE THREAD

**Issue by netravnen** [peteeckel/netbox-plugin-dns#453]
**Versions**
NetBox Version: 4.1.4
NetBox DNS Version: 1.1.4
Python Version: 3.11 (Debian Bookworm Stable)

**Describe the bug**
The bulk (CSV) importer will fail the import if the view column is omitted.

**To Reproduce**
1. Go to the record importer
2. Navigate to the data importer
3. import data with only the columns of "zone,name,type,value,status"

**Expected result**

The importer fails the task when using the headers of "zone,name,type,value,status". But works when providing the columns of "view,zone,name,type,value,status"

**Actual result**

The listed required columns are "name,name,type,value". The importer will fail, if the view column is omitted with the message "object not found".

**Code Examples**

(Requires the view to exist prior to import, view_name is case-sensitive)
```
zone,name,type,value
example.com,test1,CNAME,test.othersubdomain.example.comg
```

```
view,zone,name,type,value
Test,example.com,test1,CNAME,test.othersubdomain.example.comg
```

**Comment by netravnen** [comment]
Looked around in the code-base. Was unable to locate the function/code-block needed to be edited to require the `view` field in the bulk importer.

**Comment by peteeckel** [comment]
It's not really necessary ... 

We can apply the following logic:
* Zone names are unique -> no view needed.
* Zone names are not unique -> view needed to disambiguate.

I did that recently for a different (as of yet non-public) plugin.

**Comment by netravnen** [comment]
> * Zone names are unique -> no view needed.
> * Zone names are not unique -> view needed to disambiguate.

I did try that. Even if the zone-name is unique (within my installation). The bulk (CSV) importer will fail the import task with error "no object found <zone_column_name>"

**Comment by netravnen** [comment]
```
zone,name,type,value
zt,test1,CNAME,test.example.com
```

Sidenote: `.zt` is used by the ZeroTier VPN solution. Not relevant for the overall discussion. Just a note if anybody asks.

**Comment by netravnen** [comment]
> Requiring view generally ist clumsy as it enforces using it for importing even if the environment does not have any non-default views.

It is indeed clumsy. I do not disagree. On the other hand. It makes the use of the bulk importer painful. As the error msg presented (by Netbox) is misleading, IMHO.

**Comment by peteeckel** [comment]
I believe you that you tried ... that's why it's a bug :-)

**Comment by peteeckel** [comment]
I can import the (sliightly changed) example you provided without a view and without any problem:
```
zone,name,type,value
example.com,test1,CNAME,test.othersubdomain.example.comg
```
What error are you seeing?

**Comment by peteeckel** [comment]
```
zone,name,type,value
zt,test1,CNAME,test.example.com
```
works as well after adding the zone `zt`.

**Comment by peteeckel** [comment]
I think now I could reproduce it:
1. Define a zone `zt` in view `internal`
2. Define the same zone in view `external`(the view and zone names don't matter as long as the views are not default and the zone names are identical)
3. Now try to import 
```
zone,name,type,value
zt,test1,CNAME,test.example.com
```

Resukting error message: "Record 1 zone: Object not found: zt"

Is this correct?

**Comment by peteeckel** [comment]
Bottom line:

1. If you don't specify a view, the default view is assumed
2. If you're not working with the default view you need to specify the view name
3. The error message is ... well, it could be better. But if you can confirm the above, there is no bug.

**Comment by peteeckel** [comment]
I have now tested your examples and a couple of others manually and cannot reproduce any errors in handling imports. Your problem was probably caused by trying to refer to a zone outside the default view without specifying the view, which does not work (nor is it supposed to work). To avoid that kind of error I improved the description of the `view` field in the import form

Apart from that I improved the error messages a bit - there's unfortunately not much space for improvement, e.g. I can't refer to the name of the view within the `invalid_choice` error for a zone. 

I also added a section about importing records to the documentation.
